apps/studio/src/server/modules/site/site.service.ts

```diff
@@ -27,23 +27,35 @@ export const setSiteConfig = async (
 export const getNotification = async (siteId: number) => {
   const result = await db
     .selectFrom("Site")
-    .select(sql`config ->> 'notification'`.as("notification"))
+    .select((eb) =>
+      eb.ref("Site.config", "->").key("notification").as("notification"),
+    )
     .where("id", "=", siteId)
     .executeTakeFirstOrThrow()
```

Comment:
(out of scope, but should we fix this?) we might want to gracefully handle the error by not throwing in the DB and throw a `TRPCError` instead:

```ts
const result = await db...executeTakeFirst()
if (!result) {
  throw new TRPCError({
    code: "NOT_FOUND"
  })
}

return result.notification
```